src/store/modules/resource_stats.js

```diff
@@ -19,12 +30,26 @@ const actions = {
         content: response.data
       })
     })
+  },
+  setOrderColumn ({ commit }, value) {
```

Comment:
Why call an action first? I thought you only had to call that if the action was something that took some time or async. Otherwise you could call the mutation directly.